src/parser/shared/modules/Combatants.ts

```diff
@@ -1,31 +1,34 @@
+import { AnyEvent, HasTarget } from 'parser/core/Events';
 import Entities from './Entities';
 import Combatant from '../../core/Combatant';
 
 class Combatants extends Entities {
-  players = {};
+  players: Array<Combatant> = [];
```

Comment:
I could be wrong here, but I think this is probably best left as an object with the index as the number. Wouldn't an array fill in a bunch of `undefined`s for the unused playerIDs?

Reply:
The parent has an array defined so I had to decide between either changing this to an array or the parent to an object
https://github.com/WoWAnalyzer/WoWAnalyzer/blob/shadowlands/src/parser/shared/modules/Entities.ts#L35

So, guess I'm changing both to an object then.

Reply:
This is reminding me, during the original conversion I feel like it was weird because one of the Entities subclasses used an array and the others all used an object. If you were able to convert both to object and no tests broke then LGTM

Reply:
![image](https://user-images.githubusercontent.com/2842471/95360642-f3681500-08cb-11eb-84d6-081d556592d6.png)
everything seems to be fine